src/main/java/com/spotify/sparkey/extra/ReloadableSparkeyReader.java

```diff
@@ -109,6 +111,12 @@ private SparkeyReader createFromLogFile(File logFile) {
     }
   }
 
+  private void checkNotNull(Object o) {
```

Comment:
Duplicate of Standard API: `Objects.requireNotNull`

Reply:
Will fix